uui-components/src/pickers/PickerToggler.tsx

```diff
@@ -22,6 +22,7 @@ export interface PickerTogglerProps<TItem = any, TId = any> extends IPickerToggl
     minCharsToSearch?: number;
     prefix?: React.ReactNode;
     suffix?: React.ReactNode;
+    inputCx?: CX;
```

Comment:
here we can use just 'cx' naming and just extend IHasCX interface, because this component already specified as input.